core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -81,8 +81,6 @@ def start_navigator(self) -> None:
 
```

Comment:
same thing for service name as prefix

Reply:
Can you follow the commit style of the previous commits ?

Reply:
Done